1_basic_kfc/kubeflow.yaml

```diff
@@ -1,8 +1,11 @@
-name: xgboost4j - Train classifier
+moduleIdentifier:
+  namespace: zhizhu.com/test
+  moduleName: xgboost4j - Train classifier
+  moduleVersion: 0.0.1
 description: Trains a boosted tree ensemble classifier using xgboost4j
 
 inputs:
-- {name: Training data}
+- {name: Training data, type: DataFrameDirectory}
```

Comment:
In kubeflow spec, the `type` is optional. Should support this case.

Reply:
As discussed, let's keep type required.